tox.ini

```diff
@@ -1,111 +1,113 @@
-  AWS_DEFAULT_REGION=us-east-1
-  AWS_ACCESS_KEY_ID=accesskey
-  AWS_SECRET_ACCESS_KEY=secretkey
+    LC_ALL = en_US.utf-8
+    cdh: HADOOP_DISTRO=cdh
+    cdh: HADOOP_HOME={toxinidir}/.tox/hadoop-cdh
+    hdp: HADOOP_DISTRO=hdp
+    hdp: HADOOP_HOME={toxinidir}/.tox/hadoop-hdp
+    contrib: NOSE_ATTR=contrib
+    apache: NOSE_ATTR=apache
+    aws: NOSE_ATTR=aws
+    postgres: NOSE_ATTR=postgres
+    scheduler: NOSE_ATTR=scheduler
+    cdh,hdp: NOSE_ATTR=minicluster
+    gcloud: NOSE_ATTR=gcloud
+    unixsocket: NOSE_ATTR=unixsocket
+    dropbox: NOSE_ATTR=dropbox
+    azureblob: NOSE_ATTR=azureblob
+    core: NOSE_EVAL_ATTR=not minicluster and not gcloud and not postgres and not unixsocket and not contrib and not apache and not aws and not azureblob and not dropbox
+    LUIGI_CONFIG_PATH={toxinidir}/test/testconfig/luigi.cfg
+    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+    FULL_COVERAGE=true
+    core: NOSE_WITH_DOCTEST=1
+    AWS_DEFAULT_REGION=us-east-1
+    AWS_ACCESS_KEY_ID=accesskey
+    AWS_SECRET_ACCESS_KEY=secretkey
 commands =
-  cdh,hdp: {toxinidir}/scripts/ci/setup_hadoop_env.sh
-  azureblob: {toxinidir}/scripts/ci/install_start_azurite.sh {toxinidir}/scripts/ci
-  python --version
-  coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
-                                   {posargs:}
-  coverage combine
-  codecov -e TOXENV
-  azureblob: {toxinidir}/scripts/ci/stop_azurite.sh
+    cdh,hdp: {toxinidir}/scripts/ci/setup_hadoop_env.sh
+    azureblob: {toxinidir}/scripts/ci/install_start_azurite.sh {toxinidir}/scripts/ci
+    python --version
+    coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
+    {posargs:}
+    coverage combine
+    codecov -e TOXENV
+    azureblob: {toxinidir}/scripts/ci/stop_azurite.sh
 
 [testenv:visualiser]
 usedevelop = True
 deps =
-  mock<2.0
-  nose<2.0
-  unittest2<2.0
-  selenium==3.0.2
+    mock<2.0
+    nose<2.0
+    unittest2<2.0
+    selenium==3.0.2
 passenv = {[testenv]passenv}
 setenv =
-  LC_ALL = en_US.utf-8
-  NOSE_EVAL_ATTR=not minicluster and not gcloud and not postgres and not unixsocket
-  LUIGI_CONFIG_PATH={toxinidir}/test/testconfig/luigi.cfg
-  TEST_VISUALISER=1
+    LC_ALL = en_US.utf-8
+    NOSE_EVAL_ATTR=not minicluster and not gcloud and not postgres and not unixsocket
+    LUIGI_CONFIG_PATH={toxinidir}/test/testconfig/luigi.cfg
+    TEST_VISUALISER=1
 commands =
-  python --version
-  nosetests -v --tests=test/visualiser
+    python --version
+    nosetests -v --tests=test/visualiser
 
 # Flake8 Configuration, inspired from https://gitlab.com/pycqa/flake8/blob/master/tox.ini
 # By putting it here, local flake8 runs will also pick it up.
 [flake8]
-max-line-length=160
+max-line-length = 160
 builtins = unicode
 
 [testenv:flake8]
 deps =
-  flake8>=3.2.0
+    flake8>=3.2.0
 commands =
-  flake8 --exclude=doc,luigi/six.py,.tox
-  flake8 --max-line-length=100 --ignore=E265 doc
+    flake8 --exclude=doc,luigi/six.py,.tox,venv*
```

Comment:
`venv*` is not something that exists in this project.

Reply:
I have removed "venv*"